docs/setup/rke2-cluster.md

```diff
@@ -48,3 +48,47 @@ $ clusterctl describe cluster sample-cluster
     └─MachineDeployment/sample-cluster                    True                     22m
       └─Machine/sample-cluster-56df5b4499-zfljj           True                     23m
 ```
+
+## Deploying the Workload Cluster using SLE Micro
+
+1. Download the SLE Micro image from the [SUSE Customer Center](https://www.suse.com/download/sle-micro/). The current version supported is the 5.5 in raw format.
+
+2. Move the image downloaded into the local directory which will be defined in the environment var `OS_LOCAL_IMAGE`.
```

Comment:
I think this part should go in metal3-setup.md before we run 02_configure_host.sh?

Reply:
done! changed